templates/data/busola-ui-test-data.yaml

```diff
@@ -0,0 +1,71 @@
+templates:
+  - from: templates/generic.tmpl
+    render:
+      - to: ../prow/jobs/busola/ui-test.yaml
+        localSets:
+          extra_refs_kyma-local:
+            extra_refs:
+              local-kyma:
+                - org: "kyma-incubator"
+                  repo: "local-kyma"
+                  path_alias: "github.com/kyma-incubator/local-kyma"
+                  base_ref: "main"
+          jobConfig_default:
+            path_alias: github.com/kyma-project/busola
+          jobConfig_periodic:
+            cluster: "trusted-workload"
+          request_small:
+            request_memory: 100Mi
+            request_cpu: 50m
+          vm_job_template_k3s:
+            command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/provision-vm-and-start-busola-k3s.sh"
+          vm_job_labels_template:
+            labels:
+              preset-kyma-guard-bot-github-token: "true"
+              preset-sa-vm-kyma-integration: "true"
+              preset-gc-project-env: "true"
+        jobConfigs:
+          - repoName: kyma-project/busola
+            jobs:
+              # presubmits
+              - jobConfig:
+                  name: "pre-busola-ui-test"
+                  optional: true
+                  run_if_changed: "/"
+                  labels:
+                    preset-build-pr: "true"
+                    preset-gardener-azure-kyma-integration: "true"
+                inheritedConfigs:
+                  global:
+                    - "jobConfig_default"
+                    - "jobConfig_presubmit"
+                    - "image_kyma-integration"
+                    - "extra_refs_test-infra"
+                  local:
+                    - "jobConfig_default"
+                    - "request_small"
+                    - "vm_job_template_k3s"
+                    - "vm_job_labels_template"
+                    - "extra_refs_kyma-local"
+
+              # postsubmits
+              # - jobConfig:
```

Comment:
please add proper post submit jobs

Reply:
Do we need post submits, assuming these tests are stable?

Reply:
You're right, we might need it in the future, but for now, we don't.
But please remove the commented-out code.